src/js/hooks/list-pages/productSupplier/useProductSupplierActions.jsx

```diff
@@ -60,23 +61,27 @@ const useProductSupplierActions = ({ fireFetchData }) => {
     });
   };
 
-  const getActions = useCallback((productSupplierId) => (hasPermissionsToProductSourceActions(currentUser, isAdmin) ? [
-    {
-      defaultLabel: 'Edit',
-      label: 'react.productSupplier.edit.label',
-      leftIcon: <RiPencilLine />,
-      onClick: () => {
-        window.location = PRODUCT_SUPPLIER_URL.edit(productSupplierId);
+  const getActions = useCallback((productSupplierId) => (hasPermissions({
+    user: currentUser,
+    roles: [RoleType.ROLE_PRODUCT_MANAGER],
+    minimumRequiredRole: isAdmin,
```

Comment:
Do these need to be separated? Could `ROLE_ADMIN` be moved to the roles, and then just checked if all required roles are there?

Reply:
I think it's better to have it separated because it's a minimum required role, so when I pass the admin role, I can also be a superuser, and it still should be true. When I have to check the admin role within the roles argument I have to include the superuser here.

Reply:
@alannadolny Ok, but then call it `supplementalRoles` as it is on the backend, and maybe change the ordering of these params. So we have "required roles" first, and then "supplemental" ones.